### DISCUSSION TITLE
Is it possible to skip root domain.

### DISCUSSION THREAD

**Discussion by mrcoat2** [epi052/feroxbuster#866] [Q&A]
I have url that if you just have the domain it says the page isn't working but if you put in a path it sends a response. In the app is it possible to have it skip the plain domain and go straight to the wordlist. Sorry I'm new at this.

**Comment by TayfurYldz** [comment]
Current feroxbuster does not appear to have an option that skips the initial connectivity request to the target URL entirely. The connectivity check formats the base target itself, sends a request to it, and only drops the target on a request error; it does **not** require a 2xx response. So a root path that returns an HTTP error status such as 404/500 should still be usable as long as the HTTP request itself succeeds.

If the root path fails at the transport level but a deeper path works, the practical workaround is to make that working path the target base, e.g. `feroxbuster -u https://example.com/working-path/ -w words.txt`. Feroxbuster will append wordlist entries beneath that base path.

The relevant current logic is `FeroxScanner::connectivity` in `src/heuristics.rs`: it adds the target to `good_urls` on `Ok(_)` and skips it only on `Err(...)`. That distinction is why an HTTP 404 is different from a connection/TLS/request failure here.
